testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/soils.in

```diff
@@ -1,26 +1,33 @@
-# Soil layer definitions
-# Location: 
-#
-# depth = (cm) lower limit of layer; layers must be in order of depth.
-# matricd = (g/cm^3) bulk density of soil in this layer.
-# gravel_content = the percent volume of each layer composed of gravel (i.e., particles > 2mm)
-# evco = (frac) proportion of total baresoil evap from this layer.
-# trco = (frac) proportion of total transpiration from this layer for each vegetation type (tree, forb, shrub, grass)
-# %sand = (frac) proportion of sand in layer (0-1.0).
-# %clay = (frac) proportion of clay in layer (0-1.0).
-# imperm = (frac) proportion of 'impermeability' to water percolation(/infiltration/drainage) in layer (0-1.0)
-# soiltemp = the initial temperature of each soil layer (in celcius), from the day before the simulation starts
-# Note that the evco and trco columns must sum to 1.0 or they will
-# be normalized.
-# Soil texture data for 3 top layers from Trace's MT data. Rest of soil texture from STATSGO.
-# Note: matricd, evco, soil temp for all soil layers from Bradford & Lauenroth 2006 Appendix.
+#------ Input for soil information: soil layer, soil texture, etc.
```

Comment:
revert all changes to soils.in -- except for adding the new column %om (set to 0)